data/xml/2021.nodalida.xml

```diff
@@ -0,0 +1,537 @@
+
+% accuracy compared to the manually annotated corpus on non-standard words. Regína showed a significant improvement in accuracy when compared to an older normalization system for Icelandic. The normalized corpus and Regína will be released as open source.</abstract>
+      <url hash="64a191f0">2021.nodalida-main.45</url>
+    </paper>
+    <paper id="46">
+      <title>The <fixed-case>D</fixed-case>anish <fixed-case>G</fixed-case>igaword Corpus</title>
+      <author><first>Leon</first><last>Strømberg-Derczynski</last></author>
+      <author><first>Manuel</first><last>Ciosici</last></author>
+      <author><first>Rebekah</first><last>Baglini</last></author>
+      <author><first>Morten H.</first><last>Christiansen</last></author>
+      <author><first>Jacob Aarup</first><last>Dalsgaard</last></author>
+      <author><first>Riccardo</first><last>Fusaroli</last></author>
+      <author><first>Peter Juel</first><last>Henrichsen</last></author>
+      <author><first>Rasmus</first><last>Hvingelby</last></author>
+      <author><first>Andreas</first><last>Kirkedal</last></author>
+      <author><first>Alex Speed</first><last>Kjeldsen</last></author>
+      <author><first>Claus</first><last>Ladefoged</last></author>
+      <author><first>Finn Årup</first><last>Nielsen</last></author>
+      <author><first>Jens</first><last>Madsen</last></author>
+      <author><first>Malte Lau</first><last>Petersen</last></author>
+      <author><first>Jonathan Hvithamar</first><last>Rystrøm</last></author>
+      <author><first>Daniel</first><last>Varab</last></author>
+      <pages>413–421</pages>
+      <abstract>Danish language technology has been hindered by a lack of broad-coverage corpora at the scale modern NLP prefers. This paper describes the Danish Gigaword Corpus, the result of a focused effort to provide a diverse and freely-available one billion word corpus of Danish text. The Danish Gigaword corpus covers a wide array of time periods, domains, speakers’ socio-economic status, and Danish dialects.</abstract>
+      <url hash="ae5c62ef">2021.nodalida-main.46</url>
+    </paper>
+    <paper id="47">
+      <title><fixed-case>D</fixed-case>an<fixed-case>FEVER</fixed-case>: claim verification dataset for <fixed-case>D</fixed-case>anish</title>
+      <author><first>Jeppe</first><last>Nørregaard</last></author>
+      <author><first>Leon</first><last>Derczynski</last></author>
+      <pages>422–428</pages>
+      <abstract>We present a dataset, DanFEVER, intended for multilingual misinformation research. The dataset is in Danish and has the same format as the well-known English FEVER dataset. It can be used for testing methods in multilingual settings, as well as for creating models in production for the Danish language.</abstract>
+      <url hash="b787c42b">2021.nodalida-main.47</url>
+    </paper>
+    <paper id="48">
+      <title>The <fixed-case>I</fixed-case>celandic Word Web: A language technology-focused redesign of a lexicosemantic database</title>
```

Comment:
Word Web should be fixed-cased

Reply:
Fixed this, but didn't add it to the truest.